finality-aleph/src/network/tcp.rs

```diff
@@ -106,6 +110,14 @@ impl AddressingInformation for SignedTcpAddressingInformation {
         self.peer_id()
             .verify(&self.addressing_information.encode(), &self.signature)
     }
+
+    fn lower_level_address(&self) -> String {
+        iter::once(self.addressing_information.primary_address.clone())
+            .chain(self.addressing_information.other_addresses.clone())
+            .filter_map(|address| Some(address.parse::<SocketAddr>().ok()?.ip().to_string()))
+            .next()
+            .unwrap_or("unknown".to_string())
```

Comment:
return Option<String> instead of "unknown".

Reply:
Done in #1449